### PR TITLE
3Dplot

### PR DESCRIPTION
**Pull request by setina42**

### CODE REVIEW

File: samri/plotting/maps.py [hunk 1]
```diff
@@ -370,7 +421,230 @@ def stat(stat_maps,
...
+	plt.gca()
+	img_mesh = plt.imshow(display_3Dplot)
+	plt.show()
+	return fh
+
+def stat3D(stat_maps,
+	overlays=[],
+	figure_title="",
+	interpolation="hermite",
+	template="~/usr/shared/mouse-brain-atlases/ambmc2dsurqec_15micron_cut_mesh_1_decimated.obj",
```
Comment (TheChymera) [inline samri/plotting/maps.py]: This is not under the user home dir. This is a system-wide directory, also it is called `/usr/share`, without the “d”.

**Review by TheChymera — CHANGES_REQUESTED** [review]

**Comment by TheChymera** [comment]
@setina42 is this still in sync with the updates you made as part of https://github.com/nilearn/nilearn/pull/1869 ?

**Comment by setina42** [comment]
@TheChymera Yes, this is still in sync.

**Comment by TheChymera** [comment]
thanks :)

**Pull request merged**